bears/general/SpaceConsistencyBear.py

```diff
@@ -38,6 +56,11 @@ def run(self,
         result_texts = []
         additional_info_texts = []
 
+        if self.section.aspects and Indentation in self.section.aspects:
+            item = [aspect for aspect in self.section.aspects if
+                    isinstance(aspect, Indentation)][0]
```

Comment:
just layout candy:
put the `if` before `isinstance`, so you have
```
<element> <for x in iterable>
<if condition>
```
optional though ;)

Reply:
sure, seems logical 👍